src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -943,8 +947,13 @@ private void rawPutHelper(RawPutResponse resp) {
     }
   }
 
-  public ByteString rawPutIfAbsent(
```

Comment:
maybe we should keep this function for convenience.

Reply:
`rawPutIfAbsent` acts very different with `CompareAndSet` on `prevValue` mismatch. The former will return the existing value while the latter throws an exception. I removed `rawPutIfAbsent` because I don't want to keep two different ways to notify CAS conflict.

Reply:
As I know one of our customers is using `rawPutIfAbsent`.  If this function is removed, there may be compatibility issues. @birdstorm @Little-Wallace

Reply:
> As I know one of our customers is using `rawPutIfAbsent`.  If this function is removed, there may be compatibility issues. @birdstorm @Little-Wallace 

Right.